ChargeIo.Tests/TransactionServiceTest.cs

```diff
@@ -412,7 +412,7 @@ public void TestChargeInvalidCard()
                     Method = new CardOptions()
                     {
                         Name = "John Doe",
-                        Number = "4242424242424241",
+                        Number = "4111111111111111",
```

Comment:
This is a test case for an invalid card, but isn't this a valid test card number?

Reply:
This one ended up being pretty interesting. The original test was not designed to expect the ChargeIOException so passing a valid card number would effectively pass the test resulting in a false positive (hence my initial confusion when I browsed the code but didn't actually read the test name).

Secondly, due to the way that NUnit handles async calls we do not receive the raw ChargeIoException which further complicates matters.

It's necessary to add a check for System.AggregateException and check the inner type for the ChargeIoException when testing this case. Additionally I added a line to ensure that an exception occurs so the test will not pass if the transaction is successful.